admin_manual/configuration_server/background_jobs_configuration.rst

```diff
@@ -85,6 +85,22 @@ You can verify if the cron job has been added and scheduled by executing::
 
 .. _easyCron: https://www.easycron.com/
 
+Docker
+^^^^^^
+
+On a docker setup, you must attache the running container and invoke the cron.php.
+
+The command looks like this when your nextcloud container was names "my-nextcloud-tld"::
+
+  docker exec -ti $(docker ps  \
```

Comment:
Is the `-i` really necessary here? Since it’s cron there should be no need for interactivity

Reply:
yeah, true